app/waf/test/fixtures/datasources/testrest.datasource.ts

```diff
@@ -2186,6 +2187,33 @@ Reason:
   },
 
   installTrustedExtensions200: () => {},
+
+  bigiqLoginSucceed200: () => {
+    return {
+      token: {
+        token: 'abcdefgh',
+        exp: new Date().getTime() / 1000 + 300,
+      },
+    };
+  },
+
+  bigiqRevokeLicenseSucceed202: () => {
+    return {
+      id: '12345678',
+    };
+  },
+
+  bigiqRevokeTaskFailed200: () => {
+    return {
+      status: 'FAILED',
+    };
+  },
+
+  bigiqRevokeTaskFinished200: () => {
+    return {
+      status: 'FINISHED',
+    };
+  },
```

Comment:
Other property are useless. We can mock them once we need them.

Reply:
All the unnecessary properties of the response to you may give others important hints for problem-solving in future refining/fixing cases.

By running BIG-IQ login API, there's `token.iat`, which indicates BIG-IQ's time.
I suggest you see how `AuthToken` does to check expiration.

And more, I don't think `refreshToken` is useless. What's shortage to puting the real response here? It can show what keys in the response..

Reply:
Let's avoid to useless things.